### ISSUE TITLE
Creating a bundle for all hosts throws an exception and 500 error

### ISSUE THREAD

**Issue by dim-s** [dotCMS/core#8033]
I try to create a bundle for all my hosts but get an 500 error in my browser instead of the bundle archive. My asset folder take up much space (~3 gb). 

In my logs:

```
14:21:10,430 INFO  [stdout] (Log4J Active Asynchronous Appender) 2015-07-31 14:21:10,419 ERROR com.dotcms.publisher.ajax.RemotePublishAjaxAction - Error trying to generate bundle with id: cf3c3c7b-bcba-47f5-b2e4-e50d25d48d24
14:21:10,431 INFO  [stdout] (Log4J Active Asynchronous Appender) com.dotcms.publishing.DotBundleException: identifier is null
14:21:10,432 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.enterprise.publishing.remote.bundler.DependencyBundler.generate(SourceFile:65)
14:21:10,432 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.publisher.ajax.RemotePublishAjaxAction.generateBundle(RemotePublishAjaxAction.java:567)
14:21:10,433 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.publisher.ajax.RemotePublishAjaxAction.downloadUnpushedBundle(RemotePublishAjaxAction.java:461)
14:21:10,434 INFO  [stdout] (Log4J Active Asynchronous Appender)        at sun.reflect.GeneratedMethodAccessor1159.invoke(Unknown Source)
14:21:10,434 INFO  [stdout] (Log4J Active Asynchronous Appender)        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
14:21:10,435 INFO  [stdout] (Log4J Active Asynchronous Appender)        at java.lang.reflect.Method.invoke(Method.java:606)
14:21:10,438 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.publisher.ajax.RemotePublishAjaxAction.service(RemotePublishAjaxAction.java:124)
14:21:10,443 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.servlets.ajax.AjaxAction.doGet(AjaxAction.java:44)
14:21:10,444 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.servlets.ajax.AjaxDirectorServlet.service(AjaxDirectorServlet.java:45)
14:21:10,446 INFO  [stdout] (Log4J Active Asynchronous Appender)        at javax.servlet.http.HttpServlet.service(HttpServlet.java:847)
14:21:10,447 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:329)
14:21:10,447 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
14:21:10,448 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.filters.CMSFilter.doFilter(CMSFilter.java:228)
14:21:10,448 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:280)
14:21:10,454 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
14:21:10,454 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.filters.AutoLoginFilter.doFilter(AutoLoginFilter.java:61)
14:21:10,454 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:280)
14:21:10,456 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
14:21:10,458 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.cms.urlmap.filters.URLMapFilter.doFilter(URLMapFilter.java:299)
14:21:10,458 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:280)
14:21:10,459 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
14:21:10,459 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.filters.TimeMachineFilter.doFilter(TimeMachineFilter.java:174)
14:21:10,459 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:280)
14:21:10,460 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
14:21:10,460 INFO  [stdout] (Log4J Active Asynchronous Appender)        at nl.isaac.dotcms.mobilesite.filter.MobileUserAgentRedirectFilter.doFilter(MobileUserAgentRedirectFilter.java:78)
14:21:10,462 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:280)
14:21:10,462 INFO  [stdout] (Log4J Active Asynchronous Appender)        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)

....

14:21:10,523 INFO  [stdout] (Log4J Active Asynchronous Appender) Caused by: com.dotmarketing.business.DotStateException: identifier is null
14:21:10,523 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.business.IdentifierFactoryImpl.loadFromDb(IdentifierFactoryImpl.java:202)
14:21:10,524 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.business.IdentifierFactoryImpl.find(IdentifierFactoryImpl.java:459)
14:21:10,524 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.business.IdentifierAPIImpl.find(IdentifierAPIImpl.java:80)
14:21:10,525 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.business.VersionableFactoryImpl.getVersionInfo(VersionableFactoryImpl.java:127)
14:21:10,525 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.business.VersionableAPIImpl.getVersionInfo(VersionableAPIImpl.java:443)
14:21:10,530 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotmarketing.portlets.templates.business.TemplateAPIImpl.findWorkingTemplate(TemplateAPIImpl.java:454)
14:21:10,530 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.publisher.util.DependencyManager.setHTMLPagesDependencies(DependencyManager.java:586)
14:21:10,531 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.publisher.util.DependencyManager.setDependencies(DependencyManager.java:264)
14:21:10,531 INFO  [stdout] (Log4J Active Asynchronous Appender)        at com.dotcms.enterprise.publishing.remote.bundler.DependencyBundler.generate(SourceFile:62)
14:21:10,536 INFO  [stdout] (Log4J Active Asynchronous Appender)        ... 71 more
```

I use DotCMS 3.2.2.
